README.md

```diff
@@ -43,6 +43,20 @@ Run unit tests on your machine via the following command:
 ./gradlew test
```

Comment:
Roborazzi tests fail when I run `./gradlew test`:

> com.gravatar.gravatar.ui.components.LargeProfileSummaryTest > profileLoadingLight FAILED
>    java.lang.RuntimeException at RoboMonitoringInstrumentation.java:105

We should change the README or exclude Roborazzi tests from being ran by `./gradlew test`. I think the second option is better.

Reply:
We only have Roborazzi tests under `gravatar-ui`, so we could make a difference and tell users to run `:gravatar:test` for other unit test and `./gradlew :gravatar-ui:verifyRoborazziDebug` for checking screenshot tests.